src/api/todos.ts

```diff
@@ -5,4 +5,18 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const createTodo = (data: Omit<Todo, 'id'>) => {
+  return client.post<Todo>('/todos', data);
+};
+
+export const deleteTodo = (id: number) => {
+  return client.delete(`/todos/${id}`);
+};
+
+export const updateTodo = (id: number, data: boolean | string) => {
+  if (typeof data === 'boolean') {
+    return client.patch<Todo>(`/todos/${id}`, { completed: data });
+  }
+
+  return client.patch<Todo>(`/todos/${id}`, { title: data });
```

Comment:
It is better to separate the logic for such things, or use objects where you overwrite the necessary data

Reply:
not fixed